proofs/cbmc/freeze/Makefile

```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: Apache-2.0
+
+include ../Makefile_params.common
+
+HARNESS_ENTRY = harness
+HARNESS_FILE = freeze_harness
+
+# This should be a unique identifier for this proof, and will appear on the
+# Litani dashboard. It can be human-readable and contain spaces if you wish.
+PROOF_UID = freeze
+
+DEFINES +=
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY +=
+UNWINDSET +=
+
+PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c
+PROJECT_SOURCES += $(SRCDIR)/mldsa/reduce.c
+
+CHECK_FUNCTION_CONTRACTS=$(MLD_NAMESPACE)freeze
+USE_FUNCTION_CONTRACTS=
```

Comment:
Would it make sense to actually make use of the contracts for caddq and reduce32 as a sanity check? Or is it preferable to inline everything?

Reply:
Pushed.